Kitodo-DataFormat/src/main/java/org/kitodo/dataformat/access/Structure.java

```diff
@@ -0,0 +1,430 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.dataformat.access;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import javax.xml.bind.JAXBElement;
+import javax.xml.namespace.QName;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.kitodo.api.dataformat.mets.AreaXmlElementAccessInterface;
+import org.kitodo.api.dataformat.mets.DivXmlElementAccessInterface;
+import org.kitodo.api.dataformat.mets.MdSec;
+import org.kitodo.api.dataformat.mets.MetadataAccessInterface;
+import org.kitodo.dataformat.metskitodo.AmdSecType;
+import org.kitodo.dataformat.metskitodo.DivType;
+import org.kitodo.dataformat.metskitodo.KitodoType;
+import org.kitodo.dataformat.metskitodo.MdSecType;
+import org.kitodo.dataformat.metskitodo.MdSecType.MdWrap;
+import org.kitodo.dataformat.metskitodo.MdSecType.MdWrap.XmlData;
+import org.kitodo.dataformat.metskitodo.Mets;
+
+/**
+ * The tree-like outline structure for digital representation. This structuring
+ * structure can be subdivided into arbitrary finely granular
+ * {@link #substructures}. It can be described by {@link #metadata}.
+ */
+public class Structure implements DivXmlElementAccessInterface {
+    /**
+     * The name of the Q of Kitodo. This parameter is needed to assemble the
```

Comment:
What is Q?
![Alt Text](https://media.giphy.com/media/XqHrUf7OEa5rO/giphy.gif)

Reply:
To be honest, I have no idea. This is something that builds JAXB in memory like that and I copied it from the debugger. It works as it should.

Reply:
I finally found it: [QName](https://docs.oracle.com/javase/8/docs/api/javax/xml/namespace/QName.html)

Reply:
It would have been in the javadoc of the type, I should have just looked at it!!! (headbang)